### PR TITLE
chore: add headers

### PR DESCRIPTION
**Pull request by alexiscolin**
Add basic security headers. Feel free to add missed one (or missed properties)

### CODE REVIEW

**Comment by clockworkgr** [comment]
@kristovatlas can you review this as well?

**Comment by kristovatlas** [comment]
This looks great. Just a couple notes about Content-Security-Policy:

* `default-src: 'self'` can be used to bypass CSP if the domain hosts JSONP content, Angular.js, or user uploaded files which it probably doesn't.
* Can we set object-src to 'none' ? The object-src directive specifies a source of potentially dangerous HTML elements, such as <object>, <embed>, <applet>, <script>, <frame>, <iframe>, , <style>, <video>, <audio>, and <script>. When set to none, the object-src directive restricts the sources of these elements to none, effectively blocking all inline content from loading. This means that any attempts to load content from an external source will be blocked, and the page will only load content from trusted sources.
* Down the line, we should look into implementing [Trusted Types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/trusted-types) but I wouldn't worry about it for now.

**Comment by alexiscolin** [comment]
@kristovatlas I've added `object-src`

File: src/components/proposals/ProposalWrapper.vue [hunk 1]
```diff
@@ -708,7 +712,12 @@ onMounted(() => (title.value = `GovGen — #${proposal.value?.proposal[0].id} ${
                   <div class="grow w-1/2 mb-10">
                     <div class="text-grey-100 text-200 mb-2">{{ $t("proposalpage.labels.upgradePlan") }}</div>
                     <div class="text-light text-100">
-                      <VCodeBlock :code="JSON.stringify(proposal?.proposal[0].content.plan, null, '\t')" prismjs />
+                      bla
```
Comment (kristovatlas) [inline src/components/proposals/ProposalWrapper.vue:715]: Is this a typo?

**Comment by kristovatlas** [comment]
Looks good, just left 1 in-line comment about a possible typo @alexiscolin

File: src/components/proposals/ProposalWrapper.vue (hunk 1 above)
Comment (alexiscolin) [reply src/components/proposals/ProposalWrapper.vue]: @kristovatlas Arf yeah I pushed too fast... Thank you for your vigilance.

**Review by clockworkgr — APPROVED** [review]

**Comment by alexiscolin** [comment]
@Stuyk Do you think you could take a look and check if GH discussion works well? I wouldn't like it to be blocked for some reason at some points after the merge.

**Review by Stuyk — APPROVED** [review]

**Pull request merged**